pydicom/filereader.py

```diff
@@ -282,6 +294,12 @@ def data_element_generator(fp,
         if length != 0xFFFFFFFF:
             # don't defer loading of Specific Character Set value as it is
             # needed immediately to get the character encoding for other tags
+            if tag_set is not None and tag not in tag_set and tag != (
+                    0x08, 0x05):
+                # skip the tag if not in specific tags
+                fp.seek(fp_tell() + length)
+                continue
+
             if defer_size is not None and length > defer_size and tag != (
                     0x08, 0x05):
                 # Flag as deferred by setting value to None, and skip bytes
```

Comment:
a clarification question for me :) So these numbers (eg `0x08`) - is that hex that corresponds to the different dicom header tags? If I were reading out this particular line in English, what would it be saying?